source/jormungandr/jormungandr/new_relic.py

```diff
@@ -39,6 +40,16 @@
     agent = None
 
 
+def init(config):
+    if not agent or not config:
+        return
+    if os.path.exists(config):
+        agent.initialize(config)
+    else:
+        logging.error('%s doesn\'t exist, newrelic disabled', config)
```

Comment:
you might want use the logger from `logger = logging.getLogger(__name__)` instead ?